src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -969,6 +969,55 @@ public void testCustomPloidyRegions(final String inputFileName, final String ref
         IntegrationTestSpec.assertEqualTextFiles(output, expected);
     }
 
+    // Should fail due to non-integer values in name column of bed
+    @Test(dataProvider="HaplotypeCallerTestInputs", expectedExceptions = UserException.class)
+    public void testNonIntegerCustomPloidyRegions(final String inputFileName, final String referenceFileName) throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File output = createTempFile("output", ".vcf");
+        final File expected = new File(TEST_FILES_DIR, "expected.testPloidyRegions.vcf");
+        final File ploidyRegions = new File(TEST_FILES_DIR, "testNonIntegerPloidyRegions.bed");
+
+        final String[] args = {
+                "-I", inputFileName,
+                "-R", referenceFileName,
+                "-L", "20:10000000-10100000",
+                "-O", output.getAbsolutePath(),
+                "-ploidy", "3",
+                "--ploidy-regions", ploidyRegions.getAbsolutePath(),
+                "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false"
+        };
+
+        runCommandLine(args);
+
+        IntegrationTestSpec.assertEqualTextFiles(output, expected);
+    }
+
+    // Should fail due to negative integer values in name column of bed
+    @Test(dataProvider="HaplotypeCallerTestInputs", expectedExceptions = UserException.class)
+    public void testNonPositiveCustomPloidyRegions(final String inputFileName, final String referenceFileName) throws Exception {
```

Comment:
in keeping with my comment above... add a test here where you mis-map a bedfile with the reference as that seems detectable at the overlap detector construction phase.